### PR TITLE
[docs] who supports OT updates

### PR DESCRIPTION
**Pull request by gavinmcgovern**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ More information about Thread can be found at [threadgroup.org](http://threadgro
...
...omm" width="200px"></a><a href="https://www.samsung.com/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-samsung.png" alt="Samsung" width="200px"></a><a href="https://www.silabs.com/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-silabs.png" alt="Silicon Labs" width="200px"></a><a href="https://www.st.com/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-stm.png" alt="STMicroelectronics" width="200px"></a><a href="https://www.synopsys.com/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-synopsys.png" alt="Synopsys" width="200px"></a><a href="https://www.telink-semi.com/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-telink-github.png" alt="Telink Semiconductor" width="200px"></a><a href="https://www.ti.com/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-ti.png" alt="Texas Instruments" width="200px"></a><a href="https://www.zephyrproject.org/"><img src="https://github.com/openthread/openthread/raw/main/doc/images/ot-contrib-zephyr.png" alt="Zephyr Project" width="200px"></a>
```
Comment (Vyrastas) [inline README.md:32]: Can you make all of the new images 230x61?  So everything aligns.  Just adjust the logo to fit within each as best you can.

Comment (gavinmcgovern) [reply README.md:32]: Scaled logos now available.

Comment (Vyrastas) [inline README.md:32]: All the images need to have a canvas of exactly 230x61 or they won't align with each other on the rendered page.  These are all varying heights.

You'll have to add whitespace on top/bottom or the sides to fit them all into 230x61 as needed.

**Review by Vyrastas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gavinmcgovern) [reply README.md:32]: All logos now fit in a 230x61 canvas.

**Review by Vyrastas — CHANGES_REQUESTED** [review]
Please remove the ot-contrib-**nmb**-networks.png file (the mmb one is the correct one)

File: README.md (hunk 1 above)
Comment (Vyrastas) [inline README.md:32]: One last thing, Nanoleaf and MMB Networks should be switched to keep it alphabetical

**Review by Vyrastas — APPROVED** [review]

**Review by jwhui — APPROVED** [review]
Thanks! 👍🏼

**Pull request merged**